src/utils.tsx

```diff
@@ -413,6 +413,23 @@ export const getInitialColumnFilterFnState = <TData extends MRT_RowData>(
   return initialState;
 };
 
+interface FilterFn {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (row: MRT_RowData, id: string, filterValue: any): boolean;
+}
+
+export const customFilterFunctions: Record<string, FilterFn> = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  arrIncludesNone: (row: MRT_RowData, id: string, filterValue: any) => {
+    return !filterValue.includes(row.getValue(id));
```

Comment:
```suggestion
  (row: MRT_RowData, id: string, filterValue: string | number): boolean;
}

export const customFilterFunctions: Record<string, FilterFn> = {
  arrIncludesNone: (row: MRT_RowData, id: string, filterValue: string | number) => {
    return !filterValue.includes(row.getValue(id));
```
The type seems to be on https://www.material-react-table.com/docs/guides/column-filtering.